tests/behat/features/pa/03-report/01-102/01-report-edit-period.feature

```diff
@@ -1,5 +1,11 @@
 Feature: PA report
 
+  Scenario: PA does not see unsubmitted reports in the submitted reports section
+    Given I load the application status from "team-users-complete"
+    And I am logged in as "behat-pa1@publicguardian.gsi.gov.uk" with password "Abcd1234"
+    When I click on "pa-report-open" in the "client-1000014" region
+    Then I should see "No submitted reports" in the "client-profile-reports" region
```

Comment:
Is there already a test that assert that at least one submitted report is listed ?